OpenRA.Mods.Common/Activities/Parachute.cs

```diff
@@ -83,6 +83,7 @@ public override Activity Tick(Actor self)
 		// Only the last queued activity (given order) is kept
 		public override void Queue(Activity activity)
 		{
+			NextActivity = null;
 			NextActivity = activity;
```

Comment:
Oh, that's a left-over from when I was toying with the idea of having an assignment to `NextActivity` not override it but queue the newly assigned activity after the old one.

Fixed.